rkt/image/fetchers/common.go

```diff
@@ -0,0 +1,49 @@
+package fetchers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type InsecureOpts struct {
+	AllowHTTP      bool `json:"allow_http"`
+	SkipTLSCheck   bool `json:"skip_tls_check"`
+	SkipImageCheck bool `json:"skip_image_check"`
+}
+
+type FetcherConfig struct {
+	Scheme        string                         `json:"scheme"`
+	Name          string                         `json:"name"`
+	InsecureOpts  *InsecureOpts                  `json:"insecure"`
+	Debug         bool                           `json:"debug"`
+	Headers       map[string]map[string][]string `json:"headers"`
```

Comment:
Can we make this a type? `map[string]map[string][]string` does not tell me much ;-)